signin.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>로그인</title>
+    <link rel="stylesheet" href="css/signin_style.css">
+  </head>
+  <body>
+    <div class="content">
+      <div>
+        <a class="logo" href="index.html"><img alt="logo" src="images/logo.svg"></a>
+        <div class="sign-up">
+          회원이 아니신가요?
+          <a href="/signup.html">회원 가입하기</a>
+        </div>
+      </div>
+      <form>
+        <div class="input-email">
+          <label for="email">이메일</label>
+          <input id="email" name="email" type="email" placeholder="e-mail">
+        </div>
+        <div class="input-password">
+          <label for="password">비밀번호</label>
+          <div class="checkbox">
+            <input id="password" name="password" type="password" placeholder="비밀번호">
+            <button><img alt="blind" src="images/eye-on.svg"></button>
+          </div>
+        </div>
+        <button>로그인</button>
```

Comment:
form 태그안에 있는 input에 enter key가 입력되면 해당 form 태그와 연결된 button을 동작시킵니다.
해당 form 태그 안에는 두개의 button이 있고, 이 버튼은 제출에 해당하는 로그인 버튼이니 type='submit'을 추가하시는것이 좋겠습니다.
```suggestion
        <button type='submit'>로그인</button>
```

Reply:
mdn 에서 form 태그관련 문서를 읽어보시는걸 추천드립니다.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form